### ISSUE TITLE
jabba install 23 fails

### ISSUE THREAD

**Issue by JanecekPetr** [Jabba-Team/jabba#55]
Windows 11 23H2, Czech locale
```
> jabba install 23
Downloading 23.0.1 (https://api.foojay.io/disco/v3.0/ids/8e5d4076f95a35d195970e0e3cd1130f/redirect)
215178360/215178360
Unpacking C:\Users\PETRJA~1\AppData\Local\Temp\jabba-d-2386694289.exe to C:\Users\PetrJanecek\.jabba\jdk\23.0.1
C:\Users\PetrJanecek\.jabba\jdk\23.0.1\bin\java.exe wasn't found. If you believe this is an error - please create a ticket at https://github.com/Jabba-Team/jabba/issues (specify OS and command that was used)
```

**Comment by patrick-mccourt** [comment]
Hi @JanecekPetr, 

Are you using a managed Windows device? Seems 23.0.1 is an exe install so might prompt a UAC security prompt: 

If you're able to say yes to that it should install okay: 

![image](https://github.com/user-attachments/assets/830b74aa-faae-42f7-8360-04d10bfb8aa0)

**Comment by JanecekPetr** [comment]
I am on a managed device, yes. That's probably the issue, because when I accept the UAC prompt, I get the error mentioned above. Oh well. I'll try with some kind of sudo (either the external one, or the one on Windows 11 24H2) and report back.

**Comment by patrick-mccourt** [comment]
Do you need specifically oracle jdk 23?

jabba install openjdk@23  or temurin@23 are both zip file install so you shouldn't need UAC

**Comment by JanecekPetr** [comment]
Not at all, I've been riding on Temurin since I found this last week. I'm simply trying to get an annoying issue resolved :).

It does not work with sudo, either. I'm not sure how to see why the exe installer fails. I'll gladly help diagnose this if you let me know how.

For now we can likely park this, though, until there's more people bumping into this.
